### ISSUE TITLE
buttons or touch sensors?

### ISSUE THREAD

**Issue by ThomasWaldmann** [ecocurious2/MultiGeiger#276]
we currently have 4 pins for this option dip-switch (which is hard to reach below the esp32 module).

thinking about a more interactive or mobile usage:

can these inputs be used for?:
- buttons
- touch sensors

would be nice to control some more interactive "UI" than we have right now.

**Comment by rexfue** [comment]
They are inputs without pull up - they can be used for everything you want. But i don't think, they are good for touch sensors.

**Comment by ThomasWaldmann** [comment]
Yeah, docs say that touch is not supported at the inputs we use for the dip switch now.
